packages/components/psammead-navigation/src/index.test.jsx

```diff
@@ -9,33 +9,48 @@ import {
 import Navigation, { NavigationUl, NavigationLi } from './index';
 import igboNavData from '../testHelpers/igbo';
 
+const NavigationData = (
```

Comment:
What do you think about renaming `NavigationData` to something like `navigationUlComponent`
```jsx
<Navigation>
  {NavigationData} /* gives an impression that we are rendering a json object. */
</Navigation>
```

Reply:
I think it makes sense. Done!